src/sentry/grouping/enhancement-configs/common@2019-03-23.txt

```diff
@@ -61,3 +61,6 @@ family:native function:"?[[]SentryCrash *"                          -app -group
 family:native function:"?[[]SentryClient *"                         -app -group
 family:native function:"?[[]RNSentry *"                             -app -group
 family:native function:"?[[]SentrySDK *"                             -app -group
+
+# Java
+family:other module:*EnhancerByGuice*                          -app -group
```

Comment:
should we also ad `*FastClassByGuice*`?

https://github.com/google/guice/blob/fdfc81cde4f0dc1811489715a34b3632d9dfacd5/core/src/com/google/inject/internal/BytecodeGen.java#L69-L71

Reply:
I wasn't aware of this, good one, I'll add.